src/pymovements/dataset/dataset.py

```diff
@@ -45,6 +46,10 @@
 logger = logging.getLogger(__name__)
 
 
+@html_repr([
```

Comment:
this is needed because `Dataset` doesn't define its fields explicitly. I created  #1139 to handle this in a separate task

Reply:
Not sure I understand what you mean. It works perfectly fine if we remove the attribute list, but then the representations of some classes will just be cluttered with things that are irrelevant and not self-explanatory for most users, so I excluded them here (like `Dataset.paths`, `GazeDataFrame.n_components`, etc.).

I see these HTML representations more as a tool for beginners to explore the data structures, so I wanted to reduce them to the most important attributes.

Reply:
Ah, ok I was confused and didn't check thoroughly.

`Dataset.paths` is actually quite useful for a beginner, because currently the dataset paths aren't that well documented. It doesn't seem significant because users will seldomly define their own paths, but nevertheless important public attribute so I think it should be included.

`n_components` on the other hand I could live without in the html rep, as I plan to get rid of this attribute when implementing #453

Reply:
Okay, then we should probably also add a `_repr_html_()` for `DatasetPaths`.

Right now the `_repr_html_()` for `Dataset` without hiding any attributes looks like this:

![image](https://github.com/user-attachments/assets/6772d5c5-8964-4ca5-92e4-f5e87317cdbc)

I don't have strong opinions on which attributes to show/hide, and my choices are somewhat arbitrary (based on what I personally use often), so I'm happy to change those lists.

Reply:
included all attributes in `Dataset` and added HTML repr for `DatasetPaths`

Reply:
great!